tool/grind.dart

```diff
@@ -25,23 +24,18 @@ Directory get dartdocDocsDir =>
 Directory get sdkDocsDir => tempdirsCache.memoized1(createTempSync, 'sdkdocs');
 Directory get flutterDir => tempdirsCache.memoized1(createTempSync, 'flutter');
 
+/// Version of dartdoc we should use when making comparisons.
+String get dartdocOriginalBranch {
+  String branch = 'master';
+  if (Platform.environment.containsKey('DARTDOC_ORIGINAL')) {
```

Comment:
warn (or log?) when pulling from an env variable, so people remember they've set it?

Reply:
done.